services/api/src/routes/auth.js

```diff
@@ -72,11 +72,23 @@ router
         ctx.throw(401, 'Incorrect password');
       }
 
-      const authTokenId = generateTokenId();
-      await user.updateOne({ loginAttempts: 0, authTokenId });
+      let authTokenId = user.authTokenId;
```

Comment:
I think this is not what I want to do... each POST should generate a new token... at least maybe we should move this out from this PR?

Reply:
why would we want to reset all tokens on login?

Reply:
well for one reason it's a POST event that should cause some change. but even if not there's an expiration time that should be reset

Reply:
another reason is that it would invalidate the previous token which is better for security... I mean there's a lot of reasons frankly

Reply:
I am not following at all?
A POST has to change something ? no it doesn't.  Look at all the POST /search endpoints

I think its very uncommon to invalidate all other tokens on login? this is common on logout not login?

Reply:
not by definition but here yes a POST to a `/login` endpoint should reset the token for the reasons described above. I would assume that everyone else does this even if on the surface nothing really changes.

Reply:
kk i see the problem

Reply:
You are saying that the authTokenId can not be reused, which is fine. I assume thats like because thats the jwt standard.

But are you also saying you **cant** be logged in on 2 devices (desktop / mobile) or 2 browsers at the same time? 

This based of what I am seeing here https://github.com/bedrockio/bedrock-core/blob/master/services/api/src/utils/middleware/authenticate.js#L62